sermant-plugins/sermant-tag-transmission/tag-transmission-okhttp2.x-plugin/src/main/java/io/sermant/tag/transmission/okhttpv2/interceptors/OkHttp2xInterceptor.java

```diff
@@ -75,6 +78,17 @@ protected void injectTrafficTag2Carrier(Builder builder) {
             if (!TagKeyMatcher.isMatch(key)) {
                 continue;
             }
+
+            // if original headers contains the specific key, then ignore
+            Optional<Object> headersBuilderObject = ReflectUtils.getFieldValue(builder, "headers");
+            if (headersBuilderObject.isPresent()) {
+                Headers.Builder headersBuilder = (Headers.Builder) headersBuilderObject.get();
+                String value = headersBuilder.get(key);
+                if (value != null) {
+                    continue;
+                }
```

Comment:
Using StringUtils.isEmpty

Reply:
Empty string is permitted because it proves tag exists